### PR TITLE
feat: add ccip support to domain_to_addr

### LINKED ISSUE #72: Add CCIP support to api.starknet.id
We index the custom resolvers. We should keep a mapping for some custom resolvers to multiple URL endpoints to call to get the actual target.

We should have two ways to get this mapping:
1) We index a new event OffChainResolverUpdate which emits a list of URLs 
2) We have a hardcoded mapping in the configuration for partners using CCIP who might not have emitted that events or that gave us a non public URL endpoint (that hardcoded mapping overrides the first event)

### PR DESCRIPTION
**Pull request by irisdv**
Close #72

### CODE REVIEW

File: src/resolving.rs [hunk 1]
```diff
@@ -48,3 +54,113 @@ pub async fn get_custom_resolver(domains: &Collection<Document>, domain: &str) -
...
+                }
+            }
+        }
+        Err(err) => {
+            println!("Error while building offchain_resolver hashmap from collection offchain_resolvers: {}", err);
+        }
+    }
+}
+
+pub fn is_offchain_resolver(
```
Comment (Th0rgal) [inline src/resolving.rs]: I think get_offchain_resolver would make more sense because you get an Option and not a boolean

**Review by Th0rgal — CHANGES_REQUESTED** [review]
Great job, it's going to be very cool. Could you please add the relevant index to the indexer and may be update the docs to warn the users about the 10min warning? On this pr only a small naming change and that lgtm

**Comment by irisdv** [comment]
I updated the `update_offchain_resolvers` function to support multiple `uri`, and multiple `domains` that could have the same resolver contract.

**Review by Th0rgal — APPROVED** [review]
lgtm

**Pull request merged**